src/bde_copy_utils.cpp

```diff
@@ -990,7 +990,15 @@ int geometry_field::write_offset_geom( char *sp, data_writer *out )
 
 data_writer *file_data_writer::open( char *fname, bool append )
 {
-    FILE *f = fopen(fname,append ? "ab" : "wb" );
+    FILE *f;
+    if( strcmp(fname, "stdout") == 0)
```

Comment:
we said "-", not "stdout", or what am I missing ?

Reply:
Wasn't meant to be "stdout" have fixed.